<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="docView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>971</width>
      <height>571</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ddd;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveas"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionReplace"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="separator"/>
    <addaction name="actionLeft"/>
    <addaction name="actionCenter"/>
    <addaction name="actionRight"/>
    <addaction name="actionJust"/>
    <addaction name="separator"/>
    <addaction name="actionColor"/>
    <addaction name="actionFont"/>
    <addaction name="separator"/>
    <addaction name="actionHtml"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveas"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="separator"/>
   <addaction name="actionReplace"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionColor"/>
   <addaction name="actionFont"/>
   <addaction name="separator"/>
   <addaction name="actionLeft"/>
   <addaction name="actionCenter"/>
   <addaction name="actionRight"/>
   <addaction name="actionJust"/>
   <addaction name="separator"/>
   <addaction name="actionHtml"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/new.svg</normaloff>:/new.svg</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建一个文档</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/open.svg</normaloff>:/open.svg</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开已有文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/save.svg</normaloff>:/save.svg</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存当前文档</string>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/undo.svg</normaloff>:/undo.svg</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/redo.svg</normaloff>:/redo.svg</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="toolTip">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/copy.svg</normaloff>:/copy.svg</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制当前选中内容</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/paste.svg</normaloff>:/paste.svg</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/zoomin.svg</normaloff>:/zoomin.svg</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大视野</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/zoomout.svg</normaloff>:/zoomout.svg</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小视野</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/replace.svg</normaloff>:/replace.svg</iconset>
   </property>
   <property name="text">
    <string>查找和替换</string>
   </property>
   <property name="toolTip">
    <string>将搜索文本替换为其它内容</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>打开 </string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>替换</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/bold.png</normaloff>:/bold.png</iconset>
   </property>
   <property name="text">
    <string>加粗</string>
   </property>
   <property name="toolTip">
    <string>加粗字体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/i.png</normaloff>:/i.png</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>倾斜字体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/underline.png</normaloff>:/underline.png</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>加上下划线</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/size.png</normaloff>:/size.png</iconset>
   </property>
   <property name="text">
    <string>字体设计</string>
   </property>
   <property name="toolTip">
    <string>字体一些设计</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/cut.png</normaloff>:/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/left.png</normaloff>:/left.png</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/middle.png</normaloff>:/middle.png</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
   <property name="toolTip">
    <string>居中</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/right.png</normaloff>:/right.png</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionJust">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/just.png</normaloff>:/just.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/color.png</normaloff>:/color.png</iconset>
   </property>
   <property name="text">
    <string>颜色</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSaveas">
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/saveas.png</normaloff>:/saveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionHtml">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="img/new.qrc">
     <normaloff>:/icons8-html-filetype-24.png</normaloff>:/icons8-html-filetype-24.png</iconset>
   </property>
   <property name="text">
    <string>html切换</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img/new.qrc"/>
  <include location="img/new.qrc"/>
 </resources>
 <connections/>
</ui>
